content/400-reference/200-api-reference/250-error-reference.mdx

```diff
@@ -315,15 +315,21 @@ Possible P1012 error messages:
 
 "Transaction failed due to a write conflict or a deadlock. Please retry your transaction"
 
-### Prisma Migrate (Migration Engine)
+### Prisma Migrate (Schema engine)
+
+<Admonition type="warning">
+
+The Schema engine was previously called Migration engine. This change was introduced in version [5.0.0](https://github.com/prisma/prisma/releases/tag/5.0.0).
+
+</Admonition>
 
 #### <inlinecode>P3000</inlinecode>
 
 "Failed to create database: {database_error}"
 
 #### <inlinecode>P3001</inlinecode>
 
-"Migration possible with destructive changes and possible data loss: {migration_engine_destructive_details}"
+"Migration possible with destructive changes and possible data loss: {schema_engine_destructive_details}"
```

Comment:
Sure about this? I can not find either strings in the current codebase. Maybe this got a different term.

Reply:
I've reverted the change here